src/tests/dataLayer/databaseBulkInserter/databaseBulkInserter.test.js

```diff
@@ -123,6 +120,11 @@ describe('databaseBulkInserter Tests', () => {
                 resourceType: 'Observation',
                 doc: new Observation(observation),
             });
+            await databaseBulkInserter.insertOneAsync({
```

Comment:
I updated these to add one Consent. I still have the tests creating an Observation, too, but no Observation event will be produced.